Content.Shared/Anomaly/SharedAnomalySystem.cs

```diff
@@ -107,7 +108,8 @@ public void DoAnomalyPulse(EntityUid uid, AnomalyComponent? component = null)
         }
 
         Log.Add(LogType.Anomaly, LogImpact.Medium, $"Anomaly {ToPrettyString(uid)} pulsed with severity {component.Severity}.");
-        Audio.PlayPvs(component.PulseSound, uid);
+        if (_net.IsClient && Timing.IsFirstTimePredicted)
```

Comment:
isnt this now going to not play it on the server shouldnt it just only get played there

Reply:
electro said to do it like this so the sound would still be predicted. This will only play it on the client, not the server.

Reply:
actually i'm gonna move it to server in a desperate attempt to stop that audio bug

Reply:
Does PlayPredicted not do it?